internal/controller/volumereplicationgroup_controller.go

```diff
@@ -1963,13 +1956,11 @@ func (v *VRGInstance) updateProtectedCGsForVolRep(pvcGroups *[]ramendrv1alpha1.G
 }
 
 func (v *VRGInstance) logAndSetConditions(conditionName string, subconditions ...*metav1.Condition) {
-	msg := fmt.Sprintf("merging %s condition", conditionName)
-	v.log.Info(msg, "subconditions", subconditions)
 	finalCondition := util.MergeConditions(util.SetStatusCondition,
 		&v.instance.Status.Conditions,
 		[]string{VRGConditionReasonUnused},
 		subconditions...)
-	msg = fmt.Sprintf("updated %s status to %s", conditionName, finalCondition.Status)
+	msg := fmt.Sprintf("updated %s status to %s", conditionName, finalCondition.Status)
```

Comment:
I would get rid of this condition log printing, the i memory condition is updated to VRG once the callers are done, and so having them in the logs maybe superfluous.

There are times while inspecting logs, how the condition progressed has been useful, but given VRG status, and looking at current state of the system this log is more noise and can be removed.